modules/interactivelist.js

```diff
@@ -0,0 +1,17 @@
+const interactivelist = () => {
+  const allmarked = document.querySelectorAll('.check_box');
+  allmarked.forEach((list) => {
```

Comment:
Well done making the changes. Completed tasks are updated in localStorage. 💯 

[OPTIONAL] If I delete an item while there are completed tasks. The checked items get unchecked even though the completed property in localStorage remains `true`.

![image](https://github.com/rabayarabu/To-Do-List/assets/38088818/aaed5927-c4ae-47b2-9738-6a131b2893c5)

- It would be best to make sure that checked items remain checked on user interactions with the To-do app.